gsoc/gsoc2016/ideas_astropy.md

```diff
@@ -150,3 +150,31 @@ This project includes looking at other astronomy codes to see how they tackle th
 
 See https://github.com/OpenAstronomy/openastronomy.github.io/pull/27 for a discussion of some problems and possible solutions that will be addressed in this project.
 
+
+### Bridge sherpa and astropy fitting
+
+*Suggested Mentor(s):* D. Bourke, T. Aldcroft, H. M. Guenther
+
+*Difficulty:* Expert or better
+
+*Astronomy knowledge needed:* fitting functions and statistics
+
+*Programming skills:* Python, C, Cython
+
+#### Description
+
+
+Both astropy and Sherpa (https://github.com/sherpa/sherpa/) provide modelling and fitting capabilities; however, Sherpa's features are way
```

Comment:
.. Sherpa's features are currently more advanced, with a wider variety of built-in models, optimizers, and fit statistics.